app/invocation/invocation_model.tsx

```diff
@@ -257,7 +257,14 @@ export default class InvocationModel {
     return this.started?.command || "build";
   }
 
+  getToolName(): ToolName {
+    return this.invocations.find(() => true).role === "CI_RUNNER" ? "ci_runner" : "bazel";
```

Comment:
This is a little confusing since we have a tool concept already. Isn't this just the role?

Reply:
Replaced with role